src/XMLimport.cpp

```diff
@@ -2124,3 +2170,10 @@ void XMLimport::readIntegerList( QList<int> & list )
         }
     }
 }
+
+// This will be a string representation of a decimal float with three places of
+// decimals
+void XMLimport::getVersionString( QString & versionString )
+{
+    versionString = QString::number( static_cast<float>( mVersionMajor + 1000.0f * mVersionMinor ), 'f', 3 );
```

Comment:
:beetle: This method is not currently set to be used but yes, I got it wrong...! It should be:

    versionString = QString::number( ( mVersionMajor * 1000 + mVersionMinor ) / 1000.0, 'f', 3 );